ReleaseTooling/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -543,10 +543,15 @@ enum CocoaPodUtils {
       """)
     case let .success(output):
       let version = output.components(separatedBy: ".")
+      guard version.count >= 2 else {
+        fatalError("Failed to parse CocoaPods version: \(version)")
```

Comment:
I believe this is what is causing the occasional failure on CI - I ran into it locally once by chance.

Reply:
Hmm, I wonder how that could ever happen?